other/db/data/tables.sql

```diff
@@ -1,5 +1,3 @@
-CREATE EXTENSION IF NOT EXISTS timescaledb CASCADE;
```

Comment:
What are the implications of doing this in a current deployment? Don't we need a migration script? Same for the hypertable deletion. We also need indexes on time_detected, etc. to make sure that the performance of the current solution stays the same (or is better).